### ISSUE TITLE
Support for logits in tree classifiers for TreeSHAP-IQ

### ISSUE THREAD

**Issue by mmschlk** [mmschlk/shapiq#56]
At the moment TreeSHAP-IQ explains tree classifiers on the class-level probabilities. However it would also be important to allow for Shapley interactions to be computed for the logits.

For this also the empty prediction needs to be changed.

This may be implemented for each classifier type or more generally. Don't know exactly what this would touch.

**Comment by mmschlk** [comment]
To do this, we have to know for each model that we parse what output space it operates on ('raw', 'probabilities' or 'logits') then we can make transformations after the `TreeModel` to fit the desired output space of the user. We need to make sure that we don't mess with the `empty_prediction`. Hence, we need to re-compute the `empty_prediction` after the transformation.

**Comment by mmschlk** [comment]
I don't know if this is easily doable. In shap's TreeSHAP implementation there is an error if you do "tree-path-dependent" TreeSHAP (which is the only way to do Treeshapiq) and logits together (called log_odds in shap).
